autoload/metarw.vim

```diff
@@ -222,7 +222,7 @@ endfunction
 function! s:available_schemes()  "{{{2
   return sort(map(
   \        split(globpath(&runtimepath, 'autoload/metarw/*.vim'), "\n"),
-  \        'substitute(v:val, ''^.*/\([^/]*\)\.vim$'', ''\1'', '''')'
+  \        'substitute(v:val, ''^.*[\\/]\([^\\/]+\)\.vim$'', ''\1'', '''')'
```

Comment:
```suggestion
  \        'substitute(v:val, ''^.*[\\/]\([^\\/]*\)\.vim$'', ''\1'', '''')'
```

or use `\+` instead of `+`, because this regex does not use `\v`.

Reply:
@kana thank you, I fixed it now.